assert_debug.go

```diff
@@ -10,5 +10,9 @@ package assert
 //
 // WARN: Under the current build configuration, this assertion is enabled.
 func Debug(condition bool, msg string, values ...any) {
-	assert(condition, msg, values...)
+	assert(condition, msg, 2, values...)
+	// We tell assert() to skip 2 frames here:
+	//  1. The assert() function itself
+	//  2. This Debug() function that calls assert()
+	assert(condition, msg, 2, values...) //nolint:mnd // Explained in comment
```

Comment:
```suggestion
	// We tell assert() to skip 2 frames here:
	//  1. The assert() function itself
	//  2. This Debug() function that calls assert()
	assert(condition, msg, 2, values...) //nolint:mnd // Explained in comment
```

`assert()` gets called twice here. (Not really since the first invocation would prevent the second one, but we should remove this nonetheless). Sorry if that was my diff that caused this ✌🏻